nix/workbench/genesis/genesis.jq

```diff
@@ -6,20 +6,30 @@ def fmt_decimal_10_5($x):
 def profile_cli_args($p):
 { common:
   { createStakedArgs:
-    ([ "--supply",                 fmt_decimal_10_5($p.genesis.funds_balance)
+    ([ "--testnet-magic",          $p.genesis.network_magic
+     , "--supply",                 fmt_decimal_10_5($p.genesis.funds_balance)
      , "--gen-utxo-keys",          1
      , "--gen-genesis-keys",       $p.composition.n_bft_hosts
      , "--supply-delegated",       fmt_decimal_10_5($p.derived.supply_delegated)
      , "--gen-pools",              $p.composition.n_pools
      , "--gen-stake-delegs",       $p.derived.delegators_effective
      , "--num-stuffed-utxo",       fmt_decimal_10_5($p.derived.utxo_stuffed)
-     , "--testnet-magic",          $p.genesis.network_magic
      ] +
      if $p.composition.dense_pool_density != 1
      then
      [ "--bulk-pool-cred-files",   $p.composition.n_dense_hosts
      , "--bulk-pools-per-file",    $p.composition.dense_pool_density ]
      else [] end)
+  , createTestnetDataArgs:
+    ([ "--testnet-magic",          $p.genesis.network_magic
+     , "--total-supply",           fmt_decimal_10_5($p.genesis.funds_balance + $p.derived.supply_delegated)
```

Comment:
LGTM compared to what was being done for create-staked `--supply` and the change of semantics between `create-staked`'s `--supply` and `create-testnet-data`'s `--total-supply` :+1: